.nav

```diff
@@ -0,0 +1,4 @@
+getting-started
```

Comment:
It turns out section order in docs.yaml does not translate to section order
in output docs.

-Sandeep
Ruby Driver Dude

<http://cassandrasummit.org/Email_Signature>

On Thu, Aug 18, 2016 at 2:03 AM, Jorge Bay Gondra <notifications@github.com>
wrote:

> In .nav
> <https://github.com/datastax/nodejs-driver/pull/172#discussion_r75272084>:
>
> > @@ -0,0 +1,4 @@
> > +getting-started
>
> Is this file required? It describes the same navigation already available
> in the docs.yaml
> <https://github.com/datastax/nodejs-driver/blob/v3.1.1/docs.yaml#L5-L20>.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/datastax/nodejs-driver/pull/172/files/faa2e50c6b3919f13970dbe97424db0f0052d637#r75272084>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABGmB4tGIAtYEW4TtfEtU--FyjE_AKCFks5qhB_FgaJpZM4JnCZ4>
> .
>